scripts/management/MgmtDB.py

```diff
@@ -94,6 +96,14 @@ def update_entries_live(
                     )
                     self._insert_task(cur, realisation_name, process)
                     logger.debug("New task added to the db")
+                    #send light warnings to alert_url
+                    if alert_url != None:
+                        msg = f"fault:{entry.run_name} step:{entry.proc_type} has failed with error:{entry.error}"
+                        r = requests.post(alert_url, json={'text':msg})
+                elif (entry.status == const.Status.failed.value and self.get_retries(process, realisation_name) >= retry_max) and alert_url != None:
+                    #task failed and meet the cap or retries, send a @here to alert_url for higher attention
+                    msg = f"@here fault:{entry.run_name} step:{entry.proc_type} has failed with error:{entry.error} and meet the retry cap"
+                    r = requests.post(alert_url, json={'text':msg}) 
```

Comment:
This should be in a separate function. Preferably in a separate file.
MgmtDB shouldn't be sending internet posts.

Reply:
good point. i can move this up an level to queue_monitor